pyuvdata/uvdata/uvfits.py

```diff
@@ -973,7 +973,7 @@ def write_uvfits(
                     "UVFITS-required metadata are now set automatically to their "
                     'best known values, and no longer need to be "spoofed". As such, '
                     "the spoof_nonessential parameter is deprecated, and will be "
-                    "removed in a future release."
+                    "removed in version 2.4."
```

Comment:
Maybe worth adding to milestone list if not already.

Reply:
I just made a  new v2.4 milestone and added it there.